modules/lkqdBidAdapter.js

```diff
@@ -101,29 +61,101 @@ function buildRequests(validBidRequests) {
+        sspData.volume = bidRequest.params.volume;
+      } else {
+        sspData.volume = PARAM_VOLUME_DEFAULT;
+      }
+      if (playerWidth) {
+        sspData.width = playerWidth;
+      }
+      if (playerHeight) {
+        sspData.height = playerHeight;
+      }
+      if (bidRequest.params.hasOwnProperty('vpaidmode') && bidRequest.params.vpaidmode != null) {
+        sspData.vpaidmode = bidRequest.params.vpaidmode;
+      }
+      if (bidRequest.params.hasOwnProperty('appname') && bidRequest.params.appname != null) {
+        sspData.appname = bidRequest.params.appname;
+      }
+      if (bidRequest.params.hasOwnProperty('bundleid') && bidRequest.params.bundleid != null) {
+        sspData.bundleid = bidRequest.params.bundleid;
+      }
+      if (bidRequest.params.hasOwnProperty('aid') && bidRequest.params.aid != null) {
+        sspData.aid = bidRequest.params.aid;
+      }
+      if (bidRequest.params.hasOwnProperty('idfa') && bidRequest.params.idfa != null) {
+        sspData.idfa = bidRequest.params.idfa;
+      }
+      if (bidRequest.params.hasOwnProperty('gdpr') && bidRequest.params.gdpr != null) {
+        sspData.gdpr = bidRequest.params.gdpr;
+      }
+      if (bidRequest.params.hasOwnProperty('gdprcs') && bidRequest.params.gdprcs != null) {
+        sspData.gdprcs = bidRequest.params.gdprcs;
+      }
+      if (bidRequest.params.hasOwnProperty('flrd') && bidRequest.params.flrd != null) {
+        sspData.flrd = bidRequest.params.flrd;
+      }
+      if (bidRequest.params.hasOwnProperty('flrmp') && bidRequest.params.flrmp != null) {
+        sspData.flrmp = bidRequest.params.flrmp;
+      }
+      if (bidRequest.params.hasOwnProperty('placement') && bidRequest.params.placement != null) {
+        sspData.placement = bidRequest.params.placement;
+      }
+      if (bidRequest.params.hasOwnProperty('timeout') && bidRequest.params.timeout != null) {
+        sspData.timeout = bidRequest.params.timeout;
+      }
+      if (bidRequest.params.hasOwnProperty('dnt') && bidRequest.params.dnt != null) {
+        sspData.dnt = bidRequest.params.dnt;
+      }
+      if (bidRequest.params.hasOwnProperty('pageurl') && bidRequest.params.pageurl != null) {
+        sspData.pageurl = bidRequest.params.pageurl;
+      }
+      if (bidRequest.params.hasOwnProperty('contentId') && bidRequest.params.contentId != null) {
+        sspData.contentid = bidRequest.params.contentId;
+      }
+      if (bidRequest.params.hasOwnProperty('contentTitle') && bidRequest.params.contentTitle != null) {
+        sspData.contenttitle = bidRequest.params.contentTitle;
+      }
+      if (bidRequest.params.hasOwnProperty('contentLength') && bidRequest.params.contentLength != null) {
+        sspData.contentlength = bidRequest.params.contentLength;
+      }
+      if (bidRequest.params.hasOwnProperty('contentUrl') && bidRequest.params.contentUrl != null) {
+        sspData.contenturl = bidRequest.params.contentUrl;
+      }
+
```

Comment:
just a suggestion to make this more concise - maybe create a function to do this repetitious task.  Something like:
`function _addParam(param, default) {`
`    if (bidRequest.params.hasOwnProperty(param) && bidRequest.params[param] != null) {`
`        sspData[param] = bidRequest.params[param];`
`    } else if (default != null) {`
`        sspData[param] = default;`
`    }`
`}`

then you just repeat 
`_addParam(<bidParam>, <optionalDefault>);`
or could even put it in a loop (loop through all your params)

Reply:
This is a good suggestion - I will look into adding this improvement in a future commit but I would consider it out of scope and don't want it to hold up this current pull request.